### PR TITLE
Add Slack channel name to actions description

### PR DESCRIPTION
**Pull request by i5o**
## Describe your changes

- Added Slack channel name to the actions description of the issue, sometimes in mobile the Join button doesn't work. Having it directly linked it's an additional approach that fixes this.

## Checklist before requesting a review

- [x] I have properly bumped all version refs by running `./scripts/version-bump.sh v<new>` from project root - for example `./scripts/version-bump.sh v1.9.3`

### CODE REVIEW

**Comment by echoboomer** [comment]
@i5o would you remind rebasing this and updating the version?

**Comment by i5o** [comment]
@echoboomer done ;)

File: backend/bot/templates/incident/digest_notification.py [hunk 1]
```diff
@@ -94,7 +94,8 @@ def create(
                 "text": {
                     "type": "mrkdwn",
                     "text": "A new incident has been declared. "
-                    + "Please use the buttons here to participate.",
+                    + "Please use the buttons here to participate."
+                    + f"\n#{incident_channel_details.get("name")}",
```
Comment (echoboomer) [inline backend/bot/templates/incident/digest_notification.py:98]: You'll need to use single quotes here around `name`.

**Review by echoboomer — CHANGES_REQUESTED** [review]
Left a comment.

File: backend/bot/templates/incident/digest_notification.py (hunk 1 above)
Comment (i5o) [inline backend/bot/templates/incident/digest_notification.py:98]:
```suggestion
                    + f"\n#{incident_channel_details.get('name')}",
```

**Comment by echoboomer** [comment]
Looks like the tests just need some tweaks.

**Comment by i5o** [comment]
@echoboomer can you take a look now?

**Review by echoboomer — APPROVED** [review]

**Pull request merged**